<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWelcome</class>
 <widget class="QDialog" name="DialogWelcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bienvenue</string>
  </property>
  <widget class="QPushButton" name="pushButtonOpenLast">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir le dernier document.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir un autre document.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Creer un nouveau document.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
